tests/common.go

```diff
@@ -7019,29 +7019,120 @@ func RemoveS3BucketPolicy(bucketName string) error {
 	return nil
 }
 
+// GetAzureImmutabilityCredsFromEnv get creds for azure immutability
+func GetAzureImmutabilityCredsFromEnv() (resourceGroup, containerLevelSA, containerLevelSAKey, storageAccountLevelSA, storageAccountLevelSAKey, safeAccountLevelSA, safeAccountLevelSAKey string) {
+	resourceGroup = os.Getenv("AZURE_RESOURCE_GROUP")
+	expect(resourceGroup).NotTo(equal(""),
+		"AZURE_RESOURCE_GROUP Environment variable should not be empty")
+
+	containerLevelSA = os.Getenv("AZURE_STORAGE_ACCOUNT_NAME_CONTAINER_LEVEL")
+	expect(containerLevelSA).NotTo(equal(""),
+		"AZURE_STORAGE_ACCOUNT_NAME_CONTAINER_LEVEL Environment variable should not be empty")
+
+	containerLevelSAKey = os.Getenv("AZURE_STORAGE_ACCOUNT_KEY_CONTAINER_LEVEL")
+	expect(containerLevelSAKey).NotTo(equal(""),
+		"AZURE_STORAGE_ACCOUNT_KEY_CONTAINER_LEVEL Environment variable should not be empty")
+
+	storageAccountLevelSA = os.Getenv("AZURE_STORAGE_ACCOUNT_NAME_SA_LEVEL")
+	expect(storageAccountLevelSA).NotTo(equal(""),
+		"AZURE_STORAGE_ACCOUNT_NAME_SA_LEVEL Environment variable should not be empty")
+
+	storageAccountLevelSAKey = os.Getenv("AZURE_STORAGE_ACCOUNT_KEY_SA_LEVEL")
+	expect(storageAccountLevelSAKey).NotTo(equal(""),
+		"AZURE_STORAGE_ACCOUNT_KEY_CONTAINER_LEVEL Environment variable should not be empty")
```

Comment:
AZURE_STORAGE_ACCOUNT_KEY_SA_LEVEL Environment variable should not be empty?

Reply:
Yes we need to make sure it's not empty.